scripts/jenkins/jenkinsfiles/Jenkinsfile-Release

```diff
@@ -0,0 +1,283 @@
+// FIXME
+def NODE_LABEL = 'docker'
+// FIXME
```

Comment:
non-relevant comment ?

Reply:
It's still an open question, whether any docker machine should be able to release (I think this is the way to go), or we should release from one machine (master) only.  I have to check with Anmol, if there is a relevant reason why the releases are being executed only on master.